README.md

```diff
@@ -91,7 +103,7 @@ If you'd prefer to build from source, use [Stack](https://docs.haskellstack.org/
 `stack install` should build and compile `echidna-test` in `~/.local/bin`.
 You will need to link against libreadline and libsecp256k1 (built with recovery enabled), which should be installed with the package manager of your choosing.
 Additionally, you need to install the latest release of [libff](https://github.com/scipr-lab/libff), you can take a look to [this script](.github/scripts/install-libff.sh) used in our CI tests.
-Some linux distributions do not ship static libraries for certain things that Haskell needs, e.g. archlinux, which will cause `stack build` to fail with linking errors because we use the `-static` flag. Removing these from `package.yaml` should get everything to build if you are not looking for a static build.
+Some linux distributions do not ship static libraries for certain things that Haskell needs, e.g. ArchLinux, which will cause `stack build` to fail with linking errors because we use the `-static` flag. Removing these from `package.yaml` should get everything to build if you are not looking for a static build.
```

Comment:
linux -> Linux, also seems that Arch Linux is with space

Reply:
Done.